server/src/http/server.rs

```diff
@@ -1,14 +1,13 @@
 use async_trait::async_trait;
-use cosmicverge_shared::protocol::{self, ActivePilot};
+use cosmicverge_shared::protocol::{self, navigation::ActivePilot};
 use database::{
-    basws_server::prelude::*,
+    basws_server::{self, prelude::*},
     cosmicverge_shared::protocol::{
-        cosmic_verge_protocol_version_requirements, CosmicVergeRequest, CosmicVergeResponse,
-        OAuthProvider,
+        cosmic_verge_protocol_version_requirements, OAuthProvider, Request, Response,
     },
-    schema::{convert_db_pilots, Account, Installation, Pilot, PilotError},
+    schema::{convert_db_pilots, pilot, Account, Installation, Pilot},
 };
-use protocol::AccountPermissions;
+use protocol::Permissions;
```

Comment:
This somehow tricked the import merger -- should be included on line 2

Reply:
Done.